include/openssl/ct.h

```diff
@@ -481,7 +485,7 @@ int i2o_SCT_signature(const SCT *sct, unsigned char **out);
 * |len| should be the length of the signature in |in|.
 * Returns the number of bytes parsed, or a negative integer if an error occurs.
 */
-int o2i_SCT_signature(SCT *sct, const unsigned char **in, size_t len);
+__owur int o2i_SCT_signature(SCT *sct, const unsigned char **in, size_t len);
```

Comment:
And also, i2o/o2i is particularly misleading here because it even breaks the existing return type convention. These need to be renamed if they need to be in the public API.